diff --git a/dist/index.mjs b/dist/index.mjs
index c87bb61a779f17edde67f20103f50bde1d630983..8fc735749783d61a74fac052b22041aa8573689d 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1061,7 +1061,8 @@ var ObjectType = function(props) {
             pl: props.inspect ? indentWidth - 0.6 : 0,
             marginLeft: marginLeft,
             color: keyColor,
-            borderLeft: props.inspect ? "1px solid ".concat(borderColor) : "none"
+            borderLeft: props.inspect ? "1px solid ".concat(borderColor) : "none",
+            width: 'max-content'
         },
         children: props.inspect ? elements : !isTrap ? /*#__PURE__*/ jsx(Box, {
             component: "span",
